src/Groups/abelian_aut.jl

```diff
@@ -195,14 +203,18 @@ function defines_automorphism(G::TorQuadModule, M::ZZMatrix)
   return true
 end
 
-function Base.show(io::IO, ::MIME"text/plain", f::AutomorphismGroupElem{T}) where T<:TorQuadModule
+function Base.show(io::IO, ::MIME"text/plain", f::AutomorphismGroupElem{TorQuadModule})
   D = domain(parent(f))
   io = pretty(io)
-  println(IOContext(io, :compact => true), "Isometry of ", Lowercase(), D, " defined by")
-  print(io, Indent(), matrix(f), Dedent())
+  println(io, "Isometry of")
+  println(IOContext(io, :compact => true), Indent(), Lowercase(), D, Dedent())
+  println(io, "with matrix representation")
+  print(io, Indent())
+  show(io, MIME"text/plain"(), matrix(f))
```

Comment:
Why is the detailed printing needed here? Strictly speaking it is against our rules (but I'd be happy to be flexible about that, but for that it'd be good to know why this is beneficial here, over 
```suggestion
  show(io, matrix(f))
```
In the examples I looked at I did not see a difference. If you are aware of examples where it matters, perhaps we could add at least one of those as a test?



Otherwise, this PR looks fine to me (and again: I am not against this line, but I'd like to understand it :-) )

Reply:
Here is a difference
```
julia> m = matrix(QQ, 2, 2, [2 0; 0 4])
[2   0]
[0   4]

julia> Base.show(stdout, m)
[2 0; 0 4]
julia> Base.show(stdout, MIME"text/plain"(), m)
[2   0]
[0   4]
```

I understand your point. We have done this in the past for `ZZLat` and `TorQuadModule` for instance where most of the information, in detailed printing, is given in a matrix. For large matrices, seeing them in one line is not super helpful and we lose the informative part of the detailed printing. The same is happening for the `ZZLatWithIsom`. So currently, the following hold:
```
julia> L = integer_lattice_with_isometry(integer_lattice(; gram=QQ[2 0; 0 4]))
Integer lattice of rank 2 and degree 2
  with isometry of finite order 1
  given by
  [1   0]
  [0   1]

julia> lattice(L)
Integer lattice of rank 2 and degree 2
with gram matrix
[2   0]
[0   4]

julia> q, f = discriminant_group(L);

julia> q
Finite quadratic module
  over integer ring
Abelian group: Z/2 x Z/4
Bilinear value module: Q/Z
Quadratic value module: Q/2Z
Gram matrix quadratic form: 
[1//2      0]
[   0   1//4]

julia> f
Isometry of finite quadratic module: Z/2 x Z/4 -> Q/2Z defined by
  [1 0; 0 1]
```
So all matrices in printing are expanded except the one for the isometry on the `TorQuadModule`. As you said, strictly speaking, this is against our rules; one could argue we should modify the printing of the other objects. But as of now, I made the choice of printing the `AutomorphismGroupElem` for `TorQuadModule` and `FinGenAbGroup` in a similar way.

Reply:
OK, that makes sense to me (and as I said, I am not opposed, I just want to understand, which I now do, thanks).

Reply:
So, would it be possible to add one test which involves a matrix that is not 1x1 ? Perhaps one of the doctests could be tweaked?

Reply:
Sure, I will add more doctests.